core/types/transaction_signing.go

```diff
@@ -192,8 +192,10 @@ func (s eip2930Signer) Sender(tx *Transaction) (common.Address, error) {
 		V = new(big.Int).Sub(V, s.chainIdMul)
 		V.Sub(V, big8)
 	case AccessListTxType:
-		// ACL txs are defined to use 0 and 1 as their recovery id, add
-		// 27 to become equivalent to unprotected Homestead signatures.
+		fallthrough
+	case DynamicFeeTxType:
+		// ACL and DynamicFee txs are defined to use 0 and 1 as their recovery
```

Comment:
ACL? You mean AL? :)

Reply:
I was only copying what was already there, but yes I think AL is probably better :)